app/services/pdf_parsers/sterling_accuris_parser.rb

```diff
@@ -0,0 +1,385 @@
+module PdfParsers
```

Comment:
I like that you have separate parsers as we agreed.

What is the flow to add a new one?